src/LiveDevelopment/Agents/RemoteFunctions.js

```diff
@@ -268,16 +269,17 @@ function RemoteFunctions(experimental) {
                 "box-shadow": "0 0 1px #fff",
                 "box-sizing": "border-box"
             };
+            
+            var preferences = config.remoteHighlight.stylesToSet;
```

Comment:
You can use `Object.assign` here to simplify the merging:

``` js
var mergedStyles = Object.assign({}, stylesToSet,  config.remoteHighlight.stylesToSet)
```

and then later use `mergedStyles` instead of `stylesToSet`